QuantizedCPU: Use per-operator headers

peterbell10 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #75576
* #75575
* #75574
* #75573
* #75572
* #75571
* #75570
* #75569
* __->__ #71217
* #71216

Differential Revision: [D33949895](https://our.internmc.facebook.com/intern/diff/D33949895)

peterbell10 committed:
QuantizedCPU: Use per-operator headers

[ghstack-poisoned]

peterbell10 committed:
Update on "QuantizedCPU: Use per-operator headers"




[ghstack-poisoned]

peterbell10 committed:
Update on "QuantizedCPU: Use per-operator headers"




[ghstack-poisoned]

peterbell10 committed:
Update on "QuantizedCPU: Use per-operator headers"




[ghstack-poisoned]

peterbell10 committed:
Update on "QuantizedCPU: Use per-operator headers"




[ghstack-poisoned]

peterbell10 committed:
Update on "QuantizedCPU: Use per-operator headers"




[ghstack-poisoned]

peterbell10 committed:
Update on "QuantizedCPU: Use per-operator headers"




[ghstack-poisoned]

peterbell10 committed:
Update on "QuantizedCPU: Use per-operator headers"




[ghstack-poisoned]

peterbell10 committed:
Update on "QuantizedCPU: Use per-operator headers"




[ghstack-poisoned]

peterbell10 committed:
Update on "QuantizedCPU: Use per-operator headers"




[ghstack-poisoned]

peterbell10 committed:
Rebase and fix merge conflicts on "QuantizedCPU: Use per-operator headers"




[ghstack-poisoned]

peterbell10 committed:
Update on "QuantizedCPU: Use per-operator headers"




[ghstack-poisoned]

peterbell10 committed:
Rebase and fix merge conflicts on "QuantizedCPU: Use per-operator headers"




[ghstack-poisoned]

peterbell10 committed:
Update on "QuantizedCPU: Use per-operator headers"




[ghstack-poisoned]

malfet commented:
@malfet has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33949895).

peterbell10 committed:
Rebase on master on "QuantizedCPU: Use per-operator headers"


Differential Revision: [D33949895](https://our.internmc.facebook.com/intern/diff/D33949895)

[ghstack-poisoned]

peterbell10 committed:
Rebase and fix merge conflicts on "QuantizedCPU: Use per-operator headers"


Differential Revision: [D33949895](https://our.internmc.facebook.com/intern/diff/D33949895)

[ghstack-poisoned]

peterbell10 committed:
Update on "QuantizedCPU: Use per-operator headers"


Differential Revision: [D33949895](https://our.internmc.facebook.com/intern/diff/D33949895)

[ghstack-poisoned]

peterbell10 committed:
Rebase and fix merge conflicts on "QuantizedCPU: Use per-operator headers"


Differential Revision: [D33949895](https://our.internmc.facebook.com/intern/diff/D33949895)

[ghstack-poisoned]

peterbell10 committed:
Rebase and fix merge conflicts on "QuantizedCPU: Use per-operator headers"


Differential Revision: [D33949895](https://our.internmc.facebook.com/intern/diff/D33949895)

[ghstack-poisoned]

peterbell10 committed:
Rebase and fix merge conflicts on "QuantizedCPU: Use per-operator headers"


Differential Revision: [D33949895](https://our.internmc.facebook.com/intern/diff/D33949895)

[ghstack-poisoned]

peterbell10 committed:
Rebase and fix merge conflicts on "QuantizedCPU: Use per-operator headers"


Differential Revision: [D33949895](https://our.internmc.facebook.com/intern/diff/D33949895)

[ghstack-poisoned]

malfet commented:
@malfet has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33949895).

peterbell10 committed:
Rebase and fix merge conflicts on "QuantizedCPU: Use per-operator headers"


Differential Revision: [D33949895](https://our.internmc.facebook.com/intern/diff/D33949895)

[ghstack-poisoned]

peterbell10 committed:
Update on "QuantizedCPU: Use per-operator headers"


Differential Revision: [D33949895](https://our.internmc.facebook.com/intern/diff/D33949895)

[ghstack-poisoned]

peterbell10 committed:
Update on "QuantizedCPU: Use per-operator headers"


Differential Revision: [D33949895](https://our.internmc.facebook.com/intern/diff/D33949895)

[ghstack-poisoned]

peterbell10 committed:
Update on "QuantizedCPU: Use per-operator headers"


Differential Revision: [D33949895](https://our.internmc.facebook.com/intern/diff/D33949895)

[ghstack-poisoned]

peterbell10 committed:
Update on "QuantizedCPU: Use per-operator headers"


Differential Revision: [D33949895](https://our.internmc.facebook.com/intern/diff/D33949895)

[ghstack-poisoned]

peterbell10 committed:
Rebase on "QuantizedCPU: Use per-operator headers"


Differential Revision: [D33949895](https://our.internmc.facebook.com/intern/diff/D33949895)

[ghstack-poisoned]

peterbell10 committed:
Rebase and fix merge conflicts on "QuantizedCPU: Use per-operator headers"


Differential Revision: [D33949895](https://our.internmc.facebook.com/intern/diff/D33949895)

[ghstack-poisoned]

peterbell10 committed:
Rebase and fix merge conflicts on "QuantizedCPU: Use per-operator headers"


Differential Revision: [D33949895](https://our.internmc.facebook.com/intern/diff/D33949895)

[ghstack-poisoned]

peterbell10 committed:
Rebase and fix merge conflicts on "QuantizedCPU: Use per-operator headers"


Differential Revision: [D33949895](https://our.internmc.facebook.com/intern/diff/D33949895)

[ghstack-poisoned]

malfet commented:
@malfet has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33949895).

peterbell10 committed:
Update on "QuantizedCPU: Use per-operator headers"


Differential Revision: [D33949895](https://our.internmc.facebook.com/intern/diff/D33949895)

[ghstack-poisoned]

peterbell10 committed:
Rebase and fix merge conflicts on "QuantizedCPU: Use per-operator headers"


Differential Revision: [D33949895](https://our.internmc.facebook.com/intern/diff/D33949895)

[ghstack-poisoned]

peterbell10 committed:
Rebase and fix merge conflicts on "QuantizedCPU: Use per-operator headers"


Differential Revision: [D33949895](https://our.internmc.facebook.com/intern/diff/D33949895)

[ghstack-poisoned]

peterbell10 committed:
Rebase and fix merge conflicts on "QuantizedCPU: Use per-operator headers"


Differential Revision: [D33949895](https://our.internmc.facebook.com/intern/diff/D33949895)

[ghstack-poisoned]

peterbell10 committed:
Rebase and remove unneeded _meta.h ops headers on "QuantizedCPU: Use per-operator headers"


Differential Revision: [D33949895](https://our.internmc.facebook.com/intern/diff/D33949895)

[ghstack-poisoned]

peterbell10 committed:
Rebase and fix merge conflicts on "QuantizedCPU: Use per-operator headers"


Differential Revision: [D33949895](https://our.internmc.facebook.com/intern/diff/D33949895)

[ghstack-poisoned]

peterbell10 committed:
Update on "QuantizedCPU: Use per-operator headers"


Differential Revision: [D33949895](https://our.internmc.facebook.com/intern/diff/D33949895)

[ghstack-poisoned]

peterbell10 committed:
Update on "QuantizedCPU: Use per-operator headers"


Differential Revision: [D33949895](https://our.internmc.facebook.com/intern/diff/D33949895)

[ghstack-poisoned]

peterbell10 committed:
Rebase and fix merge conflicts on "QuantizedCPU: Use per-operator headers"


Differential Revision: [D33949895](https://our.internmc.facebook.com/intern/diff/D33949895)

[ghstack-poisoned]

peterbell10 committed:
Update on "QuantizedCPU: Use per-operator headers"


Differential Revision: [D33949895](https://our.internmc.facebook.com/intern/diff/D33949895)

[ghstack-poisoned]

peterbell10 committed:
Rebase on "QuantizedCPU: Use per-operator headers"


Differential Revision: [D33949895](https://our.internmc.facebook.com/intern/diff/D33949895)

[ghstack-poisoned]

peterbell10 committed:
Update on "QuantizedCPU: Use per-operator headers"


Differential Revision: [D33949895](https://our.internmc.facebook.com/intern/diff/D33949895)

[ghstack-poisoned]

peterbell10 committed:
Update on "QuantizedCPU: Use per-operator headers"


Differential Revision: [D33949895](https://our.internmc.facebook.com/intern/diff/D33949895)

[ghstack-poisoned]

peterbell10 committed:
Update on "QuantizedCPU: Use per-operator headers"


Differential Revision: [D33949895](https://our.internmc.facebook.com/intern/diff/D33949895)

[ghstack-poisoned]

malfet commented:
@malfet has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33949895).

malfet commented:
Added `#include <ATen/Context.h>` to one internal callsite
